server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/Bounce.java

```diff
@@ -44,7 +51,7 @@
  * A notice text can be specified, and in such case will be inserted into the
  * notification inline text.<br>
  * If the notified message has an "error message" set, it will be inserted into
- * the notification inline text. If the <code>attachStackTrace</code> init
+ * the notification inline text. If the <code>attachError</code> init
```

Comment:
We can, if we change the documentation.
Having an unknown parameter will produce an error when starting James with the parameter name.
So users can easily find which is the mailet, and follow the documentation.

This parameter (`attachStackTrace`) was an alias of (`attachError`) used in other mailets.